src/service/template-impl.js

```diff
@@ -94,29 +94,35 @@ export class BaseTemplate {
    * @protected @final
    */
   unwrap(root) {
-    let singleElement = null;
+    const unwrapped = this.forceUnwrap(root);
+    return dev().assertElement(unwrapped.length ? root : unwrapped);
```

Comment:
`isArray(unwrapped) ? root : unwrapped`

Reply:
We also want to return `root` if `unwrapped` is `[]`, no?